boa/environment.py

```diff
@@ -266,6 +266,14 @@ def __init__(self):
 
         self._contracts = {}
 
+    @property
+    def timestamp(self):
+        return self.vm.state.timestamp
+
+    @property
+    def block_number(self):
+        return self.vm.state.block_number
+
```

Comment:
not sure these should have "special" treatment relative to other vm patchables. better to do a more full rehaul if/when we address https://github.com/vyperlang/titanoboa/issues/32